File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/Go/s466963472_comments_removed.go
package main

import (
	"fmt"
)

func solve(d int) int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		y := x * x
		ans += y * d
	}
	return ans
}

func main() {
	for {
		var d int
		_, err := fmt.Scan(&d)
		if err != nil {
			break
		}
		fmt.Println(solve(d))
	}
}

// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/Java/Python/s466963472_comments_removed.py
import sys

def read():
    global d
    try:
        d = int(next(sys.stdin).strip())
        return True
    except ValueError:
        return False

def solve():
    ans = 0
    for i in range(1, 600 // d):
        x = i * d
        y = x * x
        ans += y * d
    return ans

def main():
    while read():
        print(solve())

if __name__ == "__main__":
    main()
# End of Code
